lib/fake_braintree/sinatra_app.rb

```diff
@@ -170,16 +171,11 @@ def hash_from_request_body_with_key(key)
       if FakeBraintree.decline_all_cards?
         gzipped_response(422, FakeBraintree.create_failure.to_xml(root: 'api_error_response'))
       else
-        transaction = hash_from_request_body_with_key('transaction')
+        data = hash_from_request_body_with_key('transaction')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.